examples/vite-ts/src/App.css

```diff
@@ -0,0 +1,4 @@
+#root {
+  width: 100vw;
+  height: 100vh;
```

Comment:
The `body` is already applying these styles. Do we need them here as well?

Reply:
nope 🙂